app/controllers/api/v1/targets_controller.rb

```diff
@@ -13,6 +13,12 @@ def index
         @targets = current_api_v1_user.targets
       end
 
+      def destroy
+        @target = current_api_v1_user.targets.find(params[:id])
```

Comment:
WDYT of
```ruby
current_api_v1_user.targets.find(params[:id]).destroy!

head :no_content
```

Reply:
Just becasue I don't think we need to return the record if it was deleted

Reply:
Sure, but I think `head :ok` would be more appropriate in this case, thoughts?